src/Cli.Tests/Snapshots/EndToEndTests.TestConfigGeneratedAfterAddingEntityWithoutIEnumerables.verified.txt

```diff
@@ -25,7 +25,8 @@
         Provider: StaticWebApps
       },
       Mode: Production
-    }
+    },
+    BaseRoute: /base-route
```

Comment:
This new element needs to be added to the schema file - not necessarily in the same PR

Reply:
Thanks, added.

Reply:
@ayush3797 looks like the updated schema didn't make it in latest changes.

Reply:
oops,missed that in git add!